compiler/src/iree/compiler/Codegen/Utils/Utils.cpp

```diff
@@ -92,6 +92,16 @@ Optional<IntegerAttr> getConfigIntegerAttr(
   return attr;
 }
 
+Optional<BoolAttr> getConfigBoolAttr(IREE::HAL::ExecutableTargetAttr targetAttr,
```

Comment:
We should put it to an unnamed namespace, or add `static` to it.

Reply:
Good point! But the above `getConfigStringAttr` and `getConfigIntegerAttr` are already public, and already have one non-local caller each. So I wanted this function to be consistent with that, even though it did not not already have a non-local caller. I did forget to add its declaration to `Utils.h` - fixed, thanks for catching that.